packages/core/src/index.ts

```diff
@@ -53,6 +53,8 @@ import { subscribeToAcknowledgements } from './interactions/packet/acknowledgeme
 import { PacketForwardInteraction } from './interactions/packet/forward'
 
 import { Packet } from './messages'
+import { localAddressesFirst, AddressSorter } from '@hoprnet/hopr-utils'
+import { publicAddressesFirst } from 'libp2p-utils/src/address-sort'
```

Comment:
I guess this utility is not exposed in the library's entry file?

Reply:
No, it's not. I was thinking about re-exporing it. You think it's a better idea?

Reply:
if it's not exposed by the library then we can keep it like this, I personally don't mind